test/functional/ui/float_spec.lua

```diff
@@ -56,6 +58,31 @@ describe('floating windows', function()
     eq(1000, funcs.win_getid())
   end)
 
+  it('closed immediately by autocmd #11383', function()
+    eq('Error executing lua: [string "<nvim>"]:4: Window was closed immediately',
```

Comment:
nit (and for another PR anyway): shouldn't that be s/lua/Lua?  (IIRC Lua people really care about it)

Reply:
Yes. I've tried to title-case Lua everywhere in our docs/messages, because that's more readable (`l` looks like capital `i`), and because that's how the project is named (similar to "Vim", "Nvim").

Reply:
Though I would say "Lua" is "Lua the language" or "Lua the Project" while it is perfectly fine to refer to "some lua code"  as "lua" (which this error message is about), just as we invoke it as `v:lua`, `luaeval()` etc. We don't capitalize other feature or language names in error messages.

Reply:
I (and "they") would say "some Lua code" - I'd also say "Python code".
While quickly looking at the source, there's also "Vim(lua):E5107: Error loading lua" though, where the first one is ok to be lowercase since that's the command.

The one above/here is also about the command though, isn't it?

Reply:
> While quickly looking at the source, there's also "Vim(lua):E5107: Error loading lua" though, where the first one is ok to be lowercase since that's the command. The one above/here is also about the command though, isn't it?

Well, that's just my point: we are currently using lower-case lua everywhere in error messages (except for beginning of sentence). I don't think there is anything particularly wrong with the current status-quo to need to change, though if someone wants goes through every error message and every test and changes it consistently, they can do so of course :]

Reply:
Yeah, wouldn't be too bad probably, but it is also not clear if the above refers to the `:lua` command, or "Lua code" - it just reads like the latter, but comes from/via the first probably.